crc32c.cabal

```diff
@@ -4,10 +4,10 @@ cabal-version: 1.12
 --
 -- see: https://github.com/sol/hpack
 --
--- hash: 1a4d90d4e9381a7e6ebad0ead5753fb5d408d10dca3875cc575fa3b9a83becbc
+-- hash: cb900d1f534e3f24ee23198d420b74ded026c6b1b6c69a70ca685d93c7930def
 
 name:           crc32c
-version:        0.1.0
+version:        0.2.0.0
```

Comment:
Please use 0.2.0 for simplicity.

Reply:
Four digit versions are the norm for Haskell packages, but if that's what you want, sure.